nb/pd-pandas-intro.ipynb

```diff
@@ -1016,60 +590,33 @@
     "but these are considerably more complicated to use the data frames.\n",
     "\n",
     "Data in the data frame got converted too.\n",
-    "Each column can have a different data type but somewhat the numbers\n",
+    "Each column can have different data types but somewhat the numbers\n",
     "in \"`population 2011`\" and in \"`population 2017`\" look different.\n",
-    "This is because only floating point numbers can represent `NaN`s."
+    "The reason behind this being that we cannot have `NaN`s in a `Series` with `int`egers."
```

Comment:
This one is actually wrong :)

Bools can't have NaNs either.  Otherwise they'd become microsoft bools: True, False and Maybe